### PR TITLE
Add support for disabling trailing multiline ternary operators

### PR DESCRIPTION
**Pull request by esserj**
Add support to enforce that operators lead when using multi-line ternary operators

### CODE REVIEW

**Comment by kukulich** [comment]
"Naming is hard"

After some dicuss with my colleagues we would prefer `DisableTrailingMultiLineTernaryOperatorSniff` :)

**Comment by esserj** [comment]
Hi @kukulich,

It is sure it.
But I also like the new name a lot more!
I'll adjust is shortly

**Comment by esserj** [comment]
@kukulich I've changed it, but I did notice that that are many other rules that use the term disallow
Would you guys agree that it would be more concise to use that instead?

**Comment by esserj** [comment]
No worries @kukulich.
This should do it!

Thank you for your time!

File: README.md [hunk 1]
```diff
@@ -86,6 +86,7 @@ Slevomat Coding Standard for [PHP_CodeSniffer](https://github.com/squizlabs/PHP_
  - [SlevomatCodingStandard.ControlStructures.DisallowEmpty](doc/control-structures.md#slevomatcodingstandardcontrolstructuresdisallowempty)
  - [SlevomatCodingStandard.ControlStructures.DisallowNullSafeObjectOperator](doc/control-structures.md#slevomatcodingstandardcontrolstructuresdisallownullsafeobjectoperator)
  - [SlevomatCodingStandard.ControlStructures.DisallowShortTernaryOperator](doc/control-structures.md#slevomatcodingstandardcontrolstructuresdisallowshortternaryoperator-) 🔧
+ - [SlevomatCodingStandard.ControlStructures.DisAllowTrailingMultiLineTernaryOperatorSniff](doc/control-structures.md#slevomatcodingstandardcontrolstructuresdisallowtrailingmultilineternaryoperator-) 🔧
```
Comment (kukulich) [inline README.md:89]: Typo. `Disallow` please.

File: SlevomatCodingStandard/Sniffs/ControlStructures/DisallowTrailingMultiLineTernaryOperatorSniff.php [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+use const T_OPEN_TAG_WITH_ECHO;
+use const T_SEMICOLON;
+use const T_WHITESPACE;
+
+class DisallowTrailingMultiLineTernaryOperatorSniff implements Sniff
+{
+
+	public const CODE_TRAILING_MULTI_LINE_TERNARY_OPERATOR_USED = 'TrailingMultiLineTernaryOperatorUsed';
+
+	private const TAB_INDENT = "\t";
```
Comment (kukulich) [inline SlevomatCodingStandard/Sniffs/ControlStructures/DisallowTrailingMultiLineTernaryOperatorSniff.php:27]: We can use `IndentationHelper::TAB_INDENT`

File: doc/control-structures.md [hunk 3]
```diff
@@ -152,6 +168,7 @@ Requires `new` with parentheses.
 
 Reports `new` with useless parentheses.
 
+
```
Comment (kukulich) [inline doc/control-structures.md:171]: This line should not be in the diff.

**Review by kukulich — CHANGES_REQUESTED** [review]
Please do some minor changes and add missing tests.

**Comment by kukulich** [comment]
@esserj Can you please enable access to your repository for me? I'm not able to rebase and push.

**Comment by esserj** [comment]
@kukulich "allow maintainers to edit" doesnt seems to be available for the organisation account. Not sure why. I've added you with write access. Should be the same I assume?

**Review by kukulich — APPROVED** [review]

**Comment by kukulich** [comment]
Thanks!

**Comment by esserj** [comment]
Thanks for your time @kukulich !

**Pull request merged**